### ISSUE TITLE
Dependency warnings (yarn)

### ISSUE THREAD

**Issue by AnimeAllstar** [Clariity/react-chessboard#4]
yarn keeps throwing these warnings at me after installing the package on a create-react-app typescript project (node v14.16.1).
The component still works wonderfully, anything I could do to get rid of the dependency warnings?

```
warning "react-chessboard > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react@^16.13".
warning "react-chessboard > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dnd-html5-backend@^11.1.3".
warning "react-chessboard > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dnd-touch-backend@^11.1.3".
warning "react-chessboard > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dom@^16.13".
warning "react-chessboard > react-dnd-multi-backend > react-dnd-preview@6.0.2" has incorrect peer dependency "react@^16.13.1".
warning "react-chessboard > react-dnd-multi-backend > react-dnd-preview@6.0.2" has incorrect peer dependency "react-dnd@^11.1.3".

```

**Comment by Clariity** [comment]
react-dnd-multi-backend has a major release for version 7 which is currently in alpha but will resolve the dependency warnings, once released outside of alpha I will test it and release it assuming there are no issues

https://github.com/LouisBrunner/dnd-multi-backend/blob/main/packages/react-dnd-multi-backend/package.json

**Comment by Clariity** [comment]
Update: attempted an upgrade of react-dnd-multi-backend to ^7.0.0-alpha.4 and it broke dragging on mobile. I've created an issue in the package repo to try and get the dependencies resolved on the current stable release.

https://github.com/LouisBrunner/dnd-multi-backend/issues/76

**Comment by Clariity** [comment]
Just released https://github.com/Clariity/react-chessboard/releases/tag/v1.0.5 

@AnimeAllstar are you able to check to see if this is resolved for you with that release please?

**Comment by AnimeAllstar** [comment]
I updated Node to v16.13.1. and I'm using react v17.0.2.

Just installed the update. I still get the following errors when running `yarn upgrade`.

```
warning "react-chessboard > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react@^16.13".
warning "react-chessboard > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dnd-html5-backend@^11.1.3".
warning "react-chessboard > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dnd-touch-backend@^11.1.3".
warning "react-chessboard > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dom@^16.13".
warning "react-chessboard > react-dnd-multi-backend > react-dnd-preview@6.0.2" has incorrect peer dependency "react@^16.13.1".
warning "react-chessboard > react-dnd-multi-backend > react-dnd-preview@6.0.2" has incorrect peer dependency "react-dnd@^11.1.3".
```

The warnings do not show up otherwise.

**Comment by Clariity** [comment]
Okay thank you, so the reason is because `react-dnd-multi-backend` has a peerDependency of React ^16, my issue is still raised with them but I don't see them fixing it any time soon annoyingly. I don't think there is anything more I can do unfortunately.

**Comment by Clariity** [comment]
@AnimeAllstar fixed in 1.2.6

https://github.com/Clariity/react-chessboard/pull/39
